EasyPost/Billing.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using RestSharp;
+
+namespace EasyPost
+{
+    public static class Billing
+    {
+        /// <summary>
+        ///     Delete a payment method.
+        /// </summary>
+        /// <param name="paymentMethodId">ID of the payment method to delete.</param>
+        /// <returns>Whether the request was successful or not.</returns>
+        public static async Task<bool> DeletePaymentMethod(string paymentMethodId)
+        {
+            Request request = new Request($"credit_cards/{paymentMethodId}", Method.Delete);
```

Comment:
```suggestion
            Request request = new Request($"this can be either bank account or CC/{paymentMethodId}", Method.Delete);
```

You need to check if the delete payment method is CC or bank account